### ISSUE TITLE
Setting up a SplitView within a tabview setup shows no tab icon or name for the SplitView although you can click the blank space and Navigate to it.

### ISSUE THREAD

**Issue by PSPanesar** [wix/react-native-navigation#4490]
### Issue Description

I am setting up a app which has a tab view as the root. One of the children to this tabview is a splitview component on iOS. I can't set a icon or name to appear in the tabview for this split view screen. Other screens (component, stack) have a icon and name set properly in the options BottomTab property but when i try and set the same for the split view that does not happen.

Could someone advise how to set the name and icon in a tabview for the splitview as a child to the tabview?

---
### Environment
* React Native Navigation version: 2.1.3-patch.2
* React Native version: 0.56.0
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator/iOS/Debug

**Comment by zzorba** [comment]
I did some poking around on this.

It seems like part of the problem might be that the 'splitview' options are their own side class, not in the default NavigationOptions hierarchy.  It also seems like these options aren't overriding `initWithDict` like all the others, which might explain why they aren't taking.

https://github.com/wix/react-native-navigation/blob/83f69d4effecfbaaf17af3cebdf8a03b38bfa589/lib/ios/RNNSplitViewOptions.m

It seems like this didn't quite get the full V2 treatment -- perhaps we should try to make SplitViewOptions a subfield under options (same as 'bottomTab' or 'topBar') so that it can respect the standard OO inheritance of the rest of the options.

I'm not super familiar with ObjectiveC, but just was poking around and thought this might be the cause.

**Comment by zzorba** [comment]
Digging a bit further, it seems like commit 2afbfd6e678ba7079daae30644ab7cf97891ab0c removed the use of 'applyOn' from several Options file as part of a larger refactor, but RNNSplitViewOptions was not updated?  As far as I can tell the code that consumes the options in said file is not being called anywhere.  Going to poke around a bit more -- would be happy to try to update it but I'm unsure what's intended here.

**Comment by zzorba** [comment]
Okay -- I got a proof of concept working, along with a refactor to move 'splitView' options into its own namespace key.  https://github.com/zzorba/react-native-navigation

The key seemed to be adding this code
```
- (void)willMoveToParentViewController:(UIViewController *)parent {
	if (parent) {
		[_presenter applyOptionsOnWillMoveToParentViewController:self.resolveOptions];
	}
}
```
to RNNSplitViewController.m

Now, the 'splitView' options still aren't working yet -- but they weren't working before.  I'm not quite sure how they are supposed to be folded into the 'presenter' logic.

**Comment by zzorba** [comment]
@PSPanesar looks like my PR got merged and put into the 2.8 release earlier today.  

With this I believe that the `bottomTab` settings should work again for nested SplitViews.  The `splitView` specific params are still not working, but I'll try to figure that one out next.

**Comment by PSPanesar** [comment]
Hi @zzorba Thanks so much for that. i'll give it a try. Thanks for your ongoing help to get this working as well...
